project_resource_calendar/report/calendar_event_report.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="report.paperformat" id="paperformat_calendar_event_report">
+        <field name="name">paperformat.calendar.event.report</field>
+        <field name="default" eval="True"/>
+        <field name="format">A4</field>
```

Comment:
All these specifications for the page are validated? Because some that you are specifiying are default. Also we need to remove the logo

Reply:
These:
```suggestion
        <field name="page_width">0</field>
        <field name="orientation">Portrait</field>
        <field name="margin_top">10</field>
        <field name="margin_right">5</field>
        <field name="margin_bottom">10</field>
        <field name="margin_left">5</field>
        <field name="header_line" eval="False"/>
        <field name="header_spacing">20</field>
        <field name="dpi">90</field>
```

Reply:
@eilst: Do you mean all the configurations for the printer? Please clarify me.